inheritance/src/main/java/inheritance/Shop.java

```diff
@@ -0,0 +1,54 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Shop implements Business {
```

Comment:
This class should have a separate set of tests for each method created. Need test methods for  constructor, addReview, toString, viewReviews.